core/src/main/java/smashdudes/gameplay/AttackBox.java

```diff
@@ -11,7 +11,12 @@ public class AttackBox extends CombatBox
 
     public AttackBox()
     {
-        this(new Rectangle(), new Vector2());
+        this(new Rectangle());
+    }
+
+    public AttackBox(Rectangle rect)
+    {
+        this(rect, new Vector2());
     }
```

Comment:
Extra constructor for AttackBox. Fixes a bug in box tool when using reflection for instantiation